### PR TITLE
fix: support empty argument value and escape line break

### PR DESCRIPTION
**Pull request by cd1989**
**What this PR does / why we need it**:

- Support empty argument value "" in engine
- Escape '\n' in argument

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/workflow/workload/pod/builder.go [hunk 1]
```diff
@@ -116,6 +116,8 @@ func (m *Builder) ResolveArguments() error {
 	// Escape double quotes
```
Comment (supereagle) [inline pkg/workflow/workload/pod/builder.go]: // Escape double quotes and new lines.

File: pkg/workflow/workload/pod/builder.go [hunk 2]
```diff
@@ -116,6 +116,8 @@ func (m *Builder) ResolveArguments() error {
...
 	for k, v := range parameters {
 		parameters[k] = strings.Replace(v, "\"", "\\\"", -1)
+		parameters[k] = strings.Replace(parameters[k], "\n", "\\n", -1)
+		log.Errorf("Key: %s, Value: %s", k, parameters[k])
```
Comment (supereagle) [inline pkg/workflow/workload/pod/builder.go]: log error?

Comment (cd1989) [reply pkg/workflow/workload/pod/builder.go]: Debug code, I will remove it.

**Pull request merged**